recipes/pySW4/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
 test:
   imports:
     pySW4
+  commands:
     pySW4-plot-image --help
```

Comment:
Nitpicky: there should be `-` characters at the start of these lines (and the `pySW4` line in `imports`), though it works without them.